### PR TITLE
Tensorflow 1.0.0 

### PR DESCRIPTION
**Pull request by lilleswing**
IRV Test_Overfit is still failing.  Instead of predicting all 0's it is predicting all 1's.  I'm hoping @miaecle can help me fix it today/soon.

Besides that this is ready to go.  Sadly though it isn't 0.12.0 compatible.

This work was done by running the upgrade-script available on the tensforflow repo [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/compatibility/tf_upgrade.py)

### CODE REVIEW

File: deepchem/models/tensorflow_models/lr.py [hunk 1]
```diff
@@ -172,8 +173,10 @@ def predict_proba_on_batch(self, X):
         # transfer 2D prediction tensor to 2D x n_classes(=2) 
         complimentary = np.ones(np.shape(batch_outputs))
         complimentary = complimentary - batch_outputs
-        batch_outputs = np.concatenate(
-            [complimentary, batch_outputs], axis=batch_outputs.ndim - 1)
+        batch_outputs = np.squeeze(
+            np.stack(arrays=[complimentary, batch_outputs], 
+                     axis=batch_outputs.ndim - 1))
+
```
Comment (miaecle) [inline deepchem/models/tensorflow_models/lr.py]: Would you mind recovering to the original concatenate function? The squeeze function here may cause problems when the task dimension is only 1(single task dataset like HIV)

File: deepchem/models/tensorflow_models/lr.py [hunk 2]
```diff
@@ -208,8 +211,10 @@ def predict_on_batch(self, X):
         # transfer 2D prediction tensor to 2D x n_classes(=2) 
         complimentary = np.ones(np.shape(batch_output))
         complimentary = complimentary - batch_output
-        batch_output = np.concatenate(
-            [complimentary, batch_output], axis=batch_output.ndim - 1)
+        batch_output = np.squeeze(
+            np.stack(arrays=[complimentary, batch_output],
+                     axis=batch_output.ndim - 1))
+
```
Comment (miaecle) [inline deepchem/models/tensorflow_models/lr.py]: The same here, concatenate will be better

**Comment by miaecle** [comment]
@lilleswing Sorry for the delay! I have just checked the codes, seems to be the problem of learning rate. Would you mind changing the learning rate of IRV tests(line 442, models/tests/test_overfit.py) to 0.01? This should be enough the solve the failure. Also there are some minor parts may cause problems in the change, would you mind recovering it?(if they are compatible with tf 1.0)

**Comment by rbharath** [comment]
@lilleswing It looks like this is failing due to the same weird yapf bug that's causing #369 to fail. We need to figure out what's up with the yapf :-)

**Comment by lilleswing** [comment]
@rbharath after heavily looking into this I have found

1) YAPF behaves differently when run with the dependencies of Deepchem and without.
..* That is if you install yapf in a plain python environment it will behave like your local, if you run it from an environment with rdkit as a dependency it will run it as seen in Travis.

Why this is happening is still not known to me.  Something you should do on local is 
```bash
which yapf
```

and ensure that it is running from your conda environment and not your conda root environment.

I know its not a satisfying answer, but it is what is happening.  I wasn't able to dig all the way down into the YAPF algorithm to see exactly what dependency is causing the change, but I do have two different conda environments in which yapf==0.16.0 behaves differently.

**Comment by rbharath** [comment]
@lilleswing Amazing detective work! This is the sort of subtle bug that could have burned up multiple days of developer time :-)

I'm entirely happy to just use `which yapf` to make sure we use `yapf` from the right environment for our bugfixes.

**Comment by stein2013** [comment]
Hey guys, I actually find two bugs:

one is **tf.pack()**, in tensorflow 1.0.x  **tf.pack()** had changed to **tf.stack()**.

another is **tf.concat()**, It's because of the tensorflow1.0.x has updated the parameter order of concat, 
say, change **tf.concat(A, B)** to **tf.concat(B, A)**.

change them in deepchem/nn/layers.py and deepchem/nn/model_ops.py

**Comment by lilleswing** [comment]
Closed in favor of https://github.com/deepchem/deepchem/pull/487
